gulp/tasks/test.js

```diff
@@ -0,0 +1,8 @@
+var path    = require('path'),
+    gulp    = require('gulp'),
+    mocha   = require('gulp-mocha');
+
+gulp.task('test', function (done) {
+    return gulp.src(paths.tests, { read: false })
+        .pipe(mocha());
```

Comment:
No, I left it in there, later there may be.  
So I left the structure in place.